setup.md

```diff
@@ -0,0 +1,234 @@
+
+#### Delete your vendor directory
+
+If you are using Composer v1 we recommend you delete your vendor directory **before** you migrate your project to Private Packagist.
+Composer v1 releases sometimes use data found in your vendor directory instead of fetching fresh data.
+If this happens during the initial “composer update mirrors” run it can prevent mirror and download URLs from being updated.
+Packages will then be downloaded from the old location, so you would not be using Private Packagist's more reliable downloads and CDN.
+It can also prevent notification URLs from updating, in which case your install statistics will be inaccurate.
+
+## Add VCS repositories (Git, Hg/Mercurial, SVN/Subversion)
+
+The sample composer.json file for *acme/website* contains two VCS repositories
+
+- a fork of twig/twig at `https://github.com/acme-website/twig`. This fork contains changes you need in your project only
+- a GitHub repository at `https://github.com/acme-website/custom-wordpress-bundle` which contains a private package used only in your company
+
+Without Private Packagist, every time you run `composer update` Composer loops through all tags and branches of every VCS repository to identify which versions may be installable.
+It then checks whether the branch/tag contains a valid composer.json.
+Especially for repositories with lots of tags or branches this can take a long time.
+
+You can add repositories to Private Packagist on the Packages page clicking on *Add Package -> By Url*, selecting the type *VCS* and entering its URL. For private repositories requiring authentication you need to select a credential as explained in the next section.
+
+If you add a repository hosted on GitHub, GitLab, Bitbucket or Bitbucket Server using a credential, then Private Packagist will automatically set up a webhook to get notified about new commits and versions. Therefore it is beneficial to use a credential even if the repository is publicly available. Repositories where no webhook can be set up will only be updated once every three hours.
+
+### Private VCS repository requiring authentication
+
+For Private Packagist to be able to access repositories that require authentication, e.g. a private GitHub repository, we first need to set up a credential under “Settings” -> “Stored Credentials”. Once the credential is created the repository can be added on the Packages page via “Add Package” -> “By Url”. 
+
+Note: Even though you need to select a domain when you create a credential Private Packagist will not automatically apply the credential for all http calls to that domain. Private Packagist will only suggest it where it might be useful. Therefore it is important that you explicitly select it when you add the credential or else Private Packagist will not be able to import the repository.
```

Comment:
```suggestion
Note: Even though you need to select a domain when you create a credential Private Packagist will not automatically apply the credential for all HTTP calls to that domain. Private Packagist will only suggest it where it might be useful. Therefore it is important that you explicitly select it when you add the credential or else Private Packagist will not be able to import the repository.
```